### PR TITLE
Add Chrobalt build configs

### PR DESCRIPTION
**Pull request by kaidokert**
This adds Chrobalt specific build configs into the build matrix

b/354062221

### CODE REVIEW

**Review by andrewsavage1 — COMMENTED** [review]
Looks like right now this is the same as the Chromium configs, is that true? Is it just the base that we'll add to for Chrobalt builds in the future (instead of changing the default Chromium builds we have now)?

**Comment by kaidokert** [comment]
> Looks like right now this is the same as the Chromium configs, is that true? Is it just the base that we'll add to for Chrobalt builds in the future (instead of changing the default Chromium builds we have now)?

Intentionally the same right now yes. I want to get this in place so we can start customizing from there.

**Review by isarkis — APPROVED** [review]

**Review by andrewsavage1 — APPROVED** [review]

File: .github/config/android-chrobalt-arm.json
[the remark this answers is not in the document]
Comment (kaidokert) [reply .github/config/android-chrobalt-arm.json:1]: One is unmodified Chromium Android, and the one one will be Chrobalt Android

Right this moment they are not different, but they will be in next two days

File: .github/actions/build/action.yaml
[the remark this answers is not in the document]
Comment (kaidokert) [reply .github/actions/build/action.yaml:1]: I changed platform names around

**Pull request merged**